docs/index/index-management.md

```diff
@@ -51,15 +51,15 @@ Currently, there are 3 Index implementations in CarbonData.
 
 There are two kinds of management semantic for Index.
 
-1. Automatic Refresh: Create index without `WITH DEFERRED REBUILD` in the statement, which is by default.
-2. Manual Refresh: Create index with `WITH DEFERRED REBUILD` in the statement
+1. Automatic Refresh: Create index without `WITH DEFERRED REFRESH` in the statement, which is by default.
+2. Manual Refresh: Create index with `WITH DEFERRED REFRESH` in the statement
```

Comment:
please rewrite the 1 and 2 point for manual and automatic refresh, its not completely meaningfull.

Reply:
Description is already given below

Reply:
description is there, please change the existing sentence to more meaningful

Reply:
done